docs/autogen_config.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import os.path
+from nbconvert.nbconvertapp import NbConvertApp
+
+header = """\
+Configuration options
+=====================
+
+These options can be set in ``~/.jupyter/nbconvert_config.py``, or
+at the command line when you start it.
+"""
+
+try:
+    indir = os.path.dirname(__file__)
+except NameError:
+    indir = os.getcwd()
```

Comment:
I think on RTD, the cwd will be where conf.py is, so this should be `os.path.dirname(os.getcwd())`. This is definitely something of a hack, but it's very useful to have docs on Readthedocs.